FordLabs/PeopleMover
# 159 switch emails to cdsid for users

## Issue
Resolves 159

## What was done
- [x] Add API Endpoint to add users as editors to a space
- [x] Update UI to accept CDSIDs as input for Editors
- [x] Add/use feature toggle (#perm) for updated UI
- [x] Refactor and reorganize to make switchover to new UI (without feature toggle) easier

## How to test
1. Deploy to test environment
2. Create/open a space
3. Select Share Access
4. Enter one or more CDSIDs 
5. Click Invite
Alternate paths: 
- CDSIDs can be copy/pasted in
- Invalid entries (that don't conform to CDSID format) are kept as strings, an error message is shown, and the Invite button is disabled. 
- Text entry area will expand as many CDSIDs are added

## Accessiblity Criteria
### Testing

### Manual Checks
